File: BPMOPMobile_VietNamAirLines/BPMOPMobile_VietNamAirLines/BPMOPMobile.Droid/Presenter/SharedView/SharedViewBase.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Support.V7.App;
using Android.Views;
using Android.Widget;
using BPMOPMobile.Droid.Class;

namespace BPMOPMobile.Droid.Presenter.SharedView
{
    public class SharedViewBase
    {
        public AppCompatActivity _mainAct { get; set; }
        public LayoutInflater _inflater{ get; set; }
        public CustomBaseFragment _fragment { get; set; }
        public string _fragmentTag { get; set; }
        public View _rootView { get; set; }

        public SharedViewBase(LayoutInflater _inflater, AppCompatActivity _mainAct, CustomBaseFragment _fragment, string _fragmentTag, View _rootView)
        {
            this._inflater = _inflater;
            this._mainAct = _mainAct;
            this._fragment = _fragment;
            this._fragmentTag = _fragmentTag;
            this._rootView = _rootView;
        }
        public virtual void InitializeView()
        {
            // khởi tạo view tùy vào Control
        }

    }
}

File: VNABPMOP/app/src/main/java/com/vuthao/bpmop/shareview/SharedViewBase.java
package com.vuthao.bpmop.shareview;

import android.app.Activity;
import android.view.LayoutInflater;
import android.view.View;

public class SharedViewBase {
    protected Activity mainAct;
    protected LayoutInflater inflater;
    protected String fragmentTag;
    protected View rootView;

    public SharedViewBase() {
    }

    public SharedViewBase(LayoutInflater inflater, Activity mainAct, String fragmentTag, View rootView) {
        this.inflater = inflater;
        this.mainAct = mainAct;
        this.fragmentTag = fragmentTag;
        this.rootView = rootView;
    }

    public void initializeView() { }
}
